test/extended/jobs/jobs.go

```diff
@@ -9,7 +9,7 @@ import (
 	exeutil "github.com/openshift/origin/test/extended/util"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/kubernetes/pkg/apis/batch"
+	batchv1 "k8s.io/kubernetes/pkg/apis/batch/v1"
```

Comment:
Squash with `Remove extended/v1beta1 Job test since Job is not in extensions any more`